crane/container.go

```diff
@@ -933,8 +929,10 @@ func (c *container) createArgs(cmds []string, excluded []string) []string {
 		args = append(args, "--env-file", envFile)
 	}
 	// Expose
-	for _, expose := range c.RunParams().Expose() {
-		args = append(args, "--expose", expose)
+	if !adHoc {
```

Comment:
I can't see any collision with `expose`, so why blacklist it for adhoc containers?